src/components/GlobalCSS.svelte

```diff
@@ -1,4 +1,21 @@
 <style global>
   @import "../../node_modules/normalize.css/normalize.css";
   @import "../../node_modules/bootstrap/dist/css/bootstrap-grid.min.css";
+
+  @import url("https://fonts.googleapis.com/css2?family=Red+Hat+Display:wght@400;500;700;900&display=swap");
+
+  body {
+    font-family: "Red Hat Display", Roboto, -apple-system, BlinkMacSystemFont,
+      Segoe UI, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue,
+      sans-serif;
+    font-size: 16px;
+    color: #000;
+  }
+
+  :root {
+    --content-max-width: 75rem;
```

Comment:
I suppose IE11 does not support CSS variable. `browserlist` indicates we should support it, but somehow there's no errors/warning for it. Can you double-check whether there's some problem on Svelte's tooling, or some extra setup is needed for style lints?

Reply:
We can use stylelint with https://www.npmjs.com/package/stylelint-no-unsupported-browser-features

Reply:
I will try adding the `doiuse` plugin to our postCSS preprocessor. Should I remove all CSS variables and hardcode all these values?

Reply:
I just found this: https://github.com/postcss/postcss-custom-properties
We can try including it if you think using CSS variables is cleaner than hardcoding the values.

Reply:
So, I tried using the stylelint plugin recommended by @louischan-oursky . It shows me some useful errors but for some reason it does not error on the use of CSS variables.

Reply:
It seems it is not supported by the plugin yet: https://github.com/anandthakker/doiuse/issues/90

Let's setup stylelint anyways.